src/views/Viewer.vue

```diff
@@ -26,7 +26,7 @@
 		v-if="initiated || currentFile.modal"
 		id="viewer"
 		:class="{'icon-loading': !currentFile.loaded && !currentFile.failed}"
-		:clear-view-delay="isTesting ? -1 : 5000 /* prevent cypress timeouts */"
+		:clear-view-delay="isTesting || isMobile ? -1 : undefined /* prevent cypress timeouts and disable on mobile */"
```

Comment:
```suggestion
		:clear-view-delay="(isTesting || isMobile) ? -1 : undefined /* prevent cypress timeouts and disable on mobile */"
```
To make it easier to read :grin:

Reply:
done